src/robusta/core/reporting/base.py

```diff
@@ -272,18 +272,21 @@ def __str__(self):
 
     def get_prometheus_silence_url(self, account_id: str, cluster_name: str) -> str:
         labels: Dict[str, str] = {"alertname": self.aggregation_key, "cluster": cluster_name, "account": account_id}
-        if self.subject.namespace:
-            labels["namespace"] = self.subject.namespace
+        kind: Optional[str] = self.subject.subject_type.value
+        if kind and kind == "node":
+            pass
+        else:
+            if self.subject.namespace:
+                labels["namespace"] = self.subject.namespace
 
-        if self.silence_labels and self.silence_labels.get("service"):
-            labels["service"] = self.silence_labels["service"]
+            if self.silence_labels and self.silence_labels.get("service"):
+                labels["service"] = self.silence_labels["service"]
 
-        # In prometheus job is related to the scrape target.
-        # Kubernetes jobs are stored in job_name.
-        kind: Optional[str] = self.subject.subject_type.value
-        if kind and self.subject.name:
-            kind = "job_name" if kind == "job" else kind
-            labels[kind] = self.subject.name
+            # In prometheus, job is related to the scrape target.
+            # Kubernetes jobs are stored in job_name.
+            if kind and self.subject.name:
```

Comment:
are you sure you want to indent this, 
labels[kind] = self.subject.name
wont be run for nodes

Reply:
Yes we want node alerts to be general, without a node name